### DISCUSSION TITLE
In search of a missing 7dBm of lora

### DISCUSSION THREAD

**Discussion by kjm1102** [micropython/micropython#12397] [Hardware & Peripherals]
Has anyone checked the performance of esp32 lora boards? I've run a couple of lilgo variants & never seen a receive signal strength better than -129dBm for 125 kHz bandwidth with spreading factor 12 & coding rate 5 (their specs claim -136dBm@LoRa &125Khz&SF=12&293bps). That's a 7db disparity, a lot from an antenna tinkerers point of view. Just wondering if the -136 is typical manufacturers hyperbole or if other boards are capable of achieving this?

**Comment by davefes** [comment]
Based on the cost of the test gear I had to use in doing similar tests on RF product I would be surprised if the module's RSSI were within 5-10dB of the real value.

I only use RSSI or SNR (LoRa) for comparative antenna measurements and probably even then only for ... which antenna is better.

**Comment by robert-hh** [comment]
Since a few years I run a TTN gateway based on a RPI3 and a IC880A hardware. It works well so far. I coverage tests which I made using the spreading factor 12 & coding rate 5 setting and logs I have never seen SNR values lower than -125dB. Using a Pycom Lopy board set up as TTN gateway I have reported values of down to -138dB even if that device had a worse in-door antenna. But that's just what the software reports and not related to any antenna set-up.  None of these devices is calibrated, so these are just numbers.

**Comment by kjm1102** [comment]
Does anybody know if we're supposed to add the noise figure to the sensitivity https://unsigned.io/understanding-lora-parameters/ to get achievable values? So -130 + 6 = -131?

I take the point that the rssi values are uncalibrated & unreliable near noise floors values, looks like the only way to compare different boards is to range test them till they stop receiving & see which one lasts the longest?

**Comment by davefes** [comment]
> -130 + 6 = -131?

As far as I am aware sensitivity will depend on the noise figure but they are not "added" together.

> range test them till they stop receiving & see which one lasts the longest?

That is the ultimate test.  "Stop receiving" might be a bit harsh, probably better to look at a defined error rate.  See packet error rate 1% from the datasheet.
